react/features/desktop-picker/components/DesktopPickerPane.js

```diff
@@ -77,12 +93,27 @@ class DesktopPickerPane extends Component<Props> {
                     </div>
                 );
 
+        let checkBox;
+
+        // Only display the share audio checkbox if we're on windows and on
+        // dekstop sharing tab.
+        // App window and Mac OS screen sharing doesn't work with system audio.
+        if (type === 'screen' && Platform.OS === 'windows') {
+            checkBox = (<Checkbox
+                label = { t('dialog.screenSharingAudio') }
+                name = 'share-system-audio'
+                // eslint-disable-next-line react/jsx-no-bind
+                onChange = { ({ target: { checked } }) =>
```

Comment:
can you make this an auxiliary function?

Reply:
yup, changed.